src/actions/ast.js

```diff
@@ -61,13 +61,11 @@ export function setSymbols(sourceId: SourceId) {
       return;
     }
 
-    await dispatch(
-      ({
-        type: "SET_SYMBOLS",
-        sourceId,
-        [PROMISE]: getSymbols(sourceId)
-      }: Action)
-    );
+    await dispatch({
+      type: "SET_SYMBOLS",
+      sourceId,
+      [PROMISE]: getSymbols(sourceId)
+    });
```

Comment:
I temporarily stopped checking Promise actions, I'd like to find a solution before this lands.

Reply:
👍